src/Libraries/CoreNodeModels/DynamoConvert.cs

```diff
@@ -86,6 +91,23 @@ public ConversionUnit SelectedToConversion
             }
         }
 
+        /// <summary>
+        /// Conversion factor of from unit and to to unit.
+        /// </summary>
+        [JsonProperty("ConversionFactor")]
```

Comment:
Okay, so this made me go back and compare my Swagger definition to the Flood definition.

There is only one field sent to Reach:
`'Convert Between Units': { param: 'metricValues', value: 'Values' }`

But I missed that it is actually a three part field:
`extra.metricValues = extra.selectedMetric + ';' + extra.selectedConvertFrom + ';' + extra.selectedConvertTo;`

So, I guess we need to make that Swagger node have those three fields.
Sorry for the confusion.